src/sdk/namespace_fs.js

```diff
@@ -1759,6 +1761,17 @@ class NamespaceFS {
             throw new RpcError('BAD_REQUEST', 'Bad Request');
         }
     }
+
+    _skip_version_dir(dir_key) {
+        const parts = dir_key.split('/');
+        let skip = false;
```

Comment:
I think you can just return dir_key.indexOf(`/.${HIDDEN_VERSIONS_PATH}/`) > 0, no?

Reply:
Yeah sure. Need to be bit used to these methods :)